src/Collapse/Collapse.component.tsx

```diff
@@ -68,42 +157,42 @@ export const Collapse = ({
     closedText = DEFAULT_CLOSED_TEXT,
     openedIcon,
     closedIcon,
-    theme = DEFAULT_THEME,
-    accordionHandler,
+    variant = DEFAULT_VARIANT,
+    onClick,
     hasBottomBorder = true,
     className,
     children,
 }: CollapseProps) => {
     const [open, toggleOpen] = React.useState<boolean>(isOpen);
     const IconOpened = openedIcon || <ChevronDown />;
     const IconClosed = closedIcon || <ChevronUp />;
-    // If the user provides custom openedText but not custom closedText, then use openedText for closedText as well
+
+    // If the user provides custom openedText but not custom closedText
+    // then use openedText for closedText as well
     const textClosed =
         openedText !== DEFAULT_OPENED_TEXT && closedText === DEFAULT_CLOSED_TEXT
             ? openedText
             : closedText;
 
-    // Allows for the programmatic opening/closing of the Collapse w/o a user having to click it
+    // Allows for the programmatic opening/closing of the Collapse
+    // without a user having to click it
     React.useEffect(() => {
         toggleOpen(isOpen);
     }, [isOpen]);
 
     return (
         <StyledCollapse
-            theme={theme}
+            variant={variant}
             className={classNames(
                 'anchor-collapse',
                 open && 'open',
-                theme,
+                variant,
                 !hasBottomBorder && 'no-bottom-border',
                 className
             )}
         >
-            {/** accordionHandler is used by CollapseGroup to handle closing/opening a single Collapse component. */}
             <button
-                onClick={() =>
-                    accordionHandler ? accordionHandler() : toggleOpen(!open)
-                }
+                onClick={onClick || (() => toggleOpen(!open))}
```

Comment:
@zero0Halo Sorry, I had to try :) Renaming to `onClick` makes more sense to me. Imo, this prop should have much more applicability than just the accordion.

Reply:
I am still of the opinion that event names should not also be treated as a handler. I do like how you consolidated the logic for toggleOpen though.

Reply:
I added some thoughts on this [here](https://github.com/RetailMeNot/anchor/pull/54#discussion_r296771321).